tests/Unit/Support/Constraint/IsEquivalentCssTest.php

```diff
@@ -16,6 +16,7 @@
 final class IsEquivalentCssTest extends TestCase
 {
     use CssDataProviders;
+
```

Comment:
I saw that the blank line issue with PHP-CS-Fixer is being dealt with separately.